.github/workflows/ci.yml

```diff
@@ -57,6 +57,8 @@ jobs:
         uses: trunk-io/trunk-action@v1
         with:
 
+          save-annotations: false
+
```

Comment:
Not sure why but I had to disable `save-annotations` to make CI happy.

Reply:
What was the error with  `save-annotations` enabled?

Reply:
https://github.com/eslint/eslint/actions/runs/11533071521/job/32105566286

Reply:
That looks like an error in Trunk that we should get checked out.

@det

Reply:
This is the same problem we had in the main branch before pinning `trunk-action` to v1.1.17: https://github.com/trunk-io/trunk-action/pull/266.